chain.go

```diff
@@ -15,13 +15,17 @@
 package nftables
 
 import (
+	"encoding/binary"
+	"fmt"
 	"math"
 
 	"github.com/google/nftables/binaryutil"
 	"github.com/mdlayher/netlink"
 	"golang.org/x/sys/unix"
 )
 
+var chainHeaderType = netlink.HeaderType((unix.NFNL_SUBSYS_NFTABLES << 8) | unix.NFT_MSG_NEWCHAIN)
```

Comment:
Why is this variable defined at package-level? Can you move it into chainFromMsg please?